dapr/proto/internals/v1/reminders.proto

```diff
@@ -35,3 +35,10 @@ message Reminder {
 message Reminders {
   repeated Reminder reminders = 1;
 }
+
+// TimerFiredEvent is the event that is sent to the actor when a timer fires.
+message TimerFiredEvent {
+  google.protobuf.Timestamp fireAt = 1;
```

Comment:
`fire_at`

Reply:
I will update the protos to ensure the additions use `_`

Reply:
https://github.com/cicoyle/dapr/pull/71